<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-JN2NLV6S" ignored="0" name="FirefoxTest" tests="4" failures="1" timestamp="24 Apr 2018 07:28:11 GMT" time="162.625" errors="0">
  <testcase name="InvalidLoginDetails" time="40.772" classname="Tests.LoginTests"/>
  <testcase name="BlankEmailAddressTest" time="16.77" classname="Tests.TrackNumberTests"/>
  <testcase name="BlankOrderNumberTest" time="8.695" classname="Tests.TrackNumberTests"/>
  <testcase name="BlankOrdernumberandEmailAddressTest" time="8.857" classname="Tests.TrackNumberTests">
    <failure type="java.lang.AssertionError" message="expected [Please enter a valid order numbr.] but found [Please enter a valid order number.]">
      <![CDATA[java.lang.AssertionError: expected [Please enter a valid order numbr.] but found [Please enter a valid order number.]
at pageModules.TrackOrderPage.VerifyOrderNumberMessage(TrackOrderPage.java:36)
at Tests.TrackNumberTests.BlankOrdernumberandEmailAddressTest(TrackNumberTests.java:30)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
at java.lang.Thread.run(Thread.java:745)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- BlankOrdernumberandEmailAddressTest -->
</testsuite> <!-- FirefoxTest -->
